baseui/build.gradle

```diff
@@ -19,9 +19,15 @@ android {
     }
 
     buildTypes {
+        debug {
+            resValue("string", "baseUrl", "\"https://dev.to\"")
+            buildConfigField("String", "baseUrl", "\"https://dev.to\"")
+        }
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            resValue("string", "baseUrl", "\"https://dev.to\"")
```

Comment:
Is `resValue` needed?

Reply:
The `resValue` has been added so that when in future the `baseUrl` needs to be referred from resource files, it is already available to do so.